forge-gui/res/cardsfolder/upcoming/minthara_of_the_absolute.txt

```diff
@@ -0,0 +1,12 @@
+Name:Minthara of the Absolute
+ManaCost:2 W B
+Types:Legendary Creature Elf Cleric
+PT:2/4
+K:Starting intensity:0
+T:Mode$ ChangesZone | ValidCards$ Permanent.YouCtrl | Origin$ Battlefield | Destination$ Any | TriggerZones$ Battlefield | Execute$ TrigIntensify | ActivationLimit$ 1 | TriggerDescription$ Whenever one or more permanents you control leave the battlefield, perpetually increase Minthara's intensity and the intensity of cards named CARDNAME in your hand, graveyard, and library by 1. This ability triggers only once each turn.

```

Comment:
`Minthara's intensity and the intensity of cards named CARDNAME` > `NICKNAME's intensity and the intensity of cards named Minthara of the Absolute`

when you "cards named X" don't change X into CARDNAME